src/action-sheet.scss

```diff
@@ -0,0 +1,60 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-action-sheet;
+
+.#{$block} {
+
+  // OVERLAY
+  $fd-action-sheet-overlay-color: rgba(0, 0, 0, 0.6); // Should be fade(var(--sapBlockLayer_Background), 60 or 80 (HCB|HCW))
+
+  // SPACING
+  $fd-action-sheet-mobile-padding: 0.1875rem 0.375rem;
+  $fd-action-sheet-desktop-padding: 0.25rem 0.5rem;
+
+  @include fd-reset();
+
+  padding: $fd-action-sheet-desktop-padding;
+  width: 100%;
```

Comment:
Should we add max-width and overflow behavior for list items?
![image](https://user-images.githubusercontent.com/43764832/92146205-38c98c00-edce-11ea-8227-0f4e2f7e3851.png)

Reply:
Hello @prsdthkr With current impementation of button it was impossible, I had to add `display: flex` property to `--full-width` modifier on buttons. Buttons allow to keep overflow, because of additional clickable area on `up/down`. So now it looks like that
![image](https://user-images.githubusercontent.com/26483208/92223230-739bf480-eea0-11ea-876d-06e2054cacbe.png)

Reply:
This issue is rather coming directly from button component implementation and it should be fixed there.
![image](https://user-images.githubusercontent.com/17496353/92226797-941a7d80-eea5-11ea-9e77-eb1cb2290f01.png)